.github/workflows/test-redis.yml

```diff
@@ -66,6 +66,9 @@ jobs:
                     slave3-port: 7005
                     sleep-duration: 10
 
+            -   name: Wait for Redis to Start
```

Comment:
We may not need this, i'm just tired of the inconsistency of Redis Cluster timing out 😆 So this solves that issue

Reply:
really 15 seconds? can we shrink this a little bit